assets/src/media_utils.ts

```diff
@@ -27,12 +27,35 @@ export async function getVideoStream(): Promise<MediaStream> {
     }
   }
 
-  return Promise.reject(new Error("No input devices are available"));
+  throw new Error("No input devices are available");
 }
 
 export async function getAudioStream(): Promise<MediaStream> {
-  // TODO: this should probably follow the same flow as `getVideoStream`
-  return await navigator.mediaDevices.getUserMedia({
-    audio: AUDIO_TRACK_CONSTRAINTS,
-  });
+  const constraints = {
+    video: AUDIO_TRACK_CONSTRAINTS,
```

Comment:
C'mon, it's not enough to do cmd+c and cmd+v, "video" appears at least twice here and it means this wasn't tested at all.
It would be best if we could extract the common code to a function.

Reply:
Yikes, looks like `brain.exe` has crashed. Should be better now.